checkov/arm/checks/resource/PostgreSQLServerPublicAccessDisabled.py

```diff
@@ -0,0 +1,21 @@
+from checkov.common.models.enums import CheckCategories, CheckResult
+from checkov.arm.base_resource_value_check import BaseResourceValueCheck
+
+
+class PostgreSQLServerHasPublicAccessDisabled(BaseResourceValueCheck):
+    def __init__(self):
+        name = "Ensure that PostgreSQL server disables public network access"
+        id = "CKV_AZURE_68"
+        supported_resources = ('Microsoft.DBforPostgreSQL/servers',)
+        categories = [CheckCategories.NETWORKING]
+        super().__init__(name=name, id=id, categories=categories, supported_resources=supported_resources,
+                         missing_block_result=CheckResult.FAILED)
+
+    def get_inspected_key(self):
+        return 'properties/publicNetworkAccess'
```

Comment:
Looking for `Microsoft.DBforPostgreSQL/servers` [here](https://learn.microsoft.com/en-us/azure/templates/microsoft.dbforpostgresql/servers/databases?pivots=deployment-language-arm-template), and i don't see the `publicNetworkAccess` field inside.